Add QuantConnect.Algorithm.Framework to QuantConnect.Lean.VS15 solution

#### Expected Behavior

QuantConnect.Lean.VS2015 should be built in Visual Studio 2015 with no errors.

#### Actual Behavior

When building QuantConnect.Lean.VS2015 in Visual Studio 2015, we got the following errors.

12>CSC : error CS0006: Metadata file 'C:\QuantConnect\Lean-master\Algorithm.Framework\bin\Debug\QuantConnect.Algorithm.Framework.dll' could not be found
13>------ Rebuild All started: Project: QuantConnect.Lean.Engine, Configuration: Debug Any CPU ------
13>CSC : error CS0006: Metadata file 'C:\QuantConnect\Lean-master\Algorithm.CSharp\bin\Debug\QuantConnect.Algorithm.CSharp.dll' could not be found
14>------ Rebuild All started: Project: QuantConnect.ToolBox, Configuration: Debug Any CPU ------
14>CSC : error CS0006: Metadata file 'C:\QuantConnect\Lean-master\Engine\bin\Debug\QuantConnect.Lean.Engine.dll' could not be found
15>------ Rebuild All started: Project: QuantConnect.Lean.Launcher, Configuration: Debug Any CPU ------
15>CSC : error CS0006: Metadata file 'C:\QuantConnect\Lean-master\Algorithm.CSharp\bin\Debug\QuantConnect.Algorithm.CSharp.dll' could not be found
15>CSC : error CS0006: Metadata file 'C:\QuantConnect\Lean-master\Engine\bin\Debug\QuantConnect.Lean.Engine.dll' could not be found
15>CSC : error CS0006: Metadata file 'C:\QuantConnect\Lean-master\ToolBox\bin\Debug\QuantConnect.ToolBox.exe' could not be found
========== Rebuild All: 11 succeeded, 4 failed, 0 skipped ==========

The same error happens if we do the following:
First: open Tools > Options > NuGet Package Manager and select the options for Allow NuGet to download missing packages and Automatically check for missing packages during build in Visual Studio.
Then: Tools > NuGet Package Manager> Package Manager Console: Update-Package –reinstall

#### Potential Solution

The problem could be solved by adding manually existing project QuantConnect.Algorithm.Framework which is missing in QuantConnect.Lean.VS15 solution.

#### Reproducing the Problem

Build QuantConnect.Lean.VS2015 in Visual Studio 2015.

#### System Information

Visual Studio 2015

#### Checklist
- [x] I have completely filled out this template
- [x] I have confirmed that this issue exists on the current `master` branch
- [x] I have confirmed that this is not a duplicate issue by searching [issues](https://github.com/QuantConnect/Lean/issues)
- [x] I have provided detailed steps to reproduce the issue

--- a/QuantConnect.Lean.VS15.sln
+++ b/QuantConnect.Lean.VS15.sln
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 14
-VisualStudioVersion = 14.0.25420.1
+VisualStudioVersion = 14.0.24720.0
 MinimumVisualStudioVersion = 14.0.0.0
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "QuantConnect.Lean.Launcher", "Launcher\QuantConnect.Lean.Launcher.csproj", "{09E7B916-E58B-4021-BD8B-C10B4446E226}"
 EndProject
@@ -53,6 +53,8 @@ EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "QuantConnect.VisualStudio15Plugin", "VisualStudioPlugin\QuantConnect.VisualStudio15Plugin.csproj", "{5326A9FB-0270-4647-8C43-20C5607DB530}"
 EndProject
 Project("{888888A0-9F3D-457C-B088-3A5042F75D52}") = "QuantConnect.Algorithm.PythonTools", "Algorithm.Python\QuantConnect.Algorithm.PythonTools.pyproj", "{4A5DED1F-3E34-40E2-B1F2-9A8A02195508}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "QuantConnect.Algorithm.Framework", "Algorithm.Framework\QuantConnect.Algorithm.Framework.csproj", "{75981418-7246-4B91-B136-482728E02901}"
 EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
@@ -184,14 +186,6 @@ Global
 		{AC9A142C-B485-44D7-91FF-015C22C43D05}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{AC9A142C-B485-44D7-91FF-015C22C43D05}.Release|Any CPU.Build.0 = Release|Any CPU
 		{AC9A142C-B485-44D7-91FF-015C22C43D05}.Release|x64.ActiveCfg = Release|Any CPU
-		{5326A9FB-0270-4647-8C43-20C5607DB530}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{5326A9FB-0270-4647-8C43-20C5607DB530}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{5326A9FB-0270-4647-8C43-20C5607DB530}.Debug|x64.ActiveCfg = Debug|Any CPU
-		{5326A9FB-0270-4647-8C43-20C5607DB530}.Debug|x64.Build.0 = Debug|Any CPU
-		{5326A9FB-0270-4647-8C43-20C5607DB530}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{5326A9FB-0270-4647-8C43-20C5607DB530}.Release|Any CPU.Build.0 = Release|Any CPU
-		{5326A9FB-0270-4647-8C43-20C5607DB530}.Release|x64.ActiveCfg = Release|Any CPU
-		{5326A9FB-0270-4647-8C43-20C5607DB530}.Release|x64.Build.0 = Release|Any CPU
 		{9561D14A-467E-40AD-928E-EE9F758D7D98}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{9561D14A-467E-40AD-928E-EE9F758D7D98}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{9561D14A-467E-40AD-928E-EE9F758D7D98}.Debug|x64.ActiveCfg = Debug|Any CPU
@@ -200,10 +194,26 @@ Global
 		{9561D14A-467E-40AD-928E-EE9F758D7D98}.Release|Any CPU.Build.0 = Release|Any CPU
 		{9561D14A-467E-40AD-928E-EE9F758D7D98}.Release|x64.ActiveCfg = Release|Any CPU
 		{9561D14A-467E-40AD-928E-EE9F758D7D98}.Release|x64.Build.0 = Release|Any CPU
+		{5326A9FB-0270-4647-8C43-20C5607DB530}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{5326A9FB-0270-4647-8C43-20C5607DB530}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{5326A9FB-0270-4647-8C43-20C5607DB530}.Debug|x64.ActiveCfg = Debug|Any CPU
+		{5326A9FB-0270-4647-8C43-20C5607DB530}.Debug|x64.Build.0 = Debug|Any CPU
+		{5326A9FB-0270-4647-8C43-20C5607DB530}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{5326A9FB-0270-4647-8C43-20C5607DB530}.Release|Any CPU.Build.0 = Release|Any CPU
+		{5326A9FB-0270-4647-8C43-20C5607DB530}.Release|x64.ActiveCfg = Release|Any CPU
+		{5326A9FB-0270-4647-8C43-20C5607DB530}.Release|x64.Build.0 = Release|Any CPU
 		{4A5DED1F-3E34-40E2-B1F2-9A8A02195508}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{4A5DED1F-3E34-40E2-B1F2-9A8A02195508}.Debug|x64.ActiveCfg = Debug|Any CPU
 		{4A5DED1F-3E34-40E2-B1F2-9A8A02195508}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{4A5DED1F-3E34-40E2-B1F2-9A8A02195508}.Release|x64.ActiveCfg = Release|Any CPU
+		{75981418-7246-4B91-B136-482728E02901}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{75981418-7246-4B91-B136-482728E02901}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{75981418-7246-4B91-B136-482728E02901}.Debug|x64.ActiveCfg = Debug|Any CPU
+		{75981418-7246-4B91-B136-482728E02901}.Debug|x64.Build.0 = Debug|Any CPU
+		{75981418-7246-4B91-B136-482728E02901}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{75981418-7246-4B91-B136-482728E02901}.Release|Any CPU.Build.0 = Release|Any CPU
+		{75981418-7246-4B91-B136-482728E02901}.Release|x64.ActiveCfg = Release|Any CPU
+		{75981418-7246-4B91-B136-482728E02901}.Release|x64.Build.0 = Release|Any CPU
 	EndGlobalSection
 	GlobalSection(SolutionProperties) = preSolution
 		HideSolutionNode = FALSE

